### PR TITLE
feat: [SRM-9907]: addressing demo feedback

### PR DESCRIPTION
**Pull request by harshilbits**
##### Summary:
Addressing demo feedback
1. Health score -> service Health
2. SLI at the top and preselected.
3. SLA -> 99.9%.

##### Jira Links:
1. https://harness.atlassian.net/browse/SRM-9907

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Review by DFieldFL — DISMISSED** [review]

**Review by vardanbansal-harness — DISMISSED** [review]

**Comment by harshilbits** [comment]
trigger cypress

File: src/modules/85-cv/pages/health-source/common/SelectHealthSourceServices/SelectHealthSourceServices.tsx [hunk 1]
```diff
@@ -29,20 +29,21 @@ export default function SelectHealthSourceServices({
         <Text tooltipProps={{ dataTooltipId: 'assignLabel' }} className={css.groupLabel}>
           {getString('cv.monitoredServices.assignLabel')}
         </Text>
-
+        {!hideSLIAndHealthScore ? (
```
Comment (deepesh-ui) [inline src/modules/85-cv/pages/health-source/common/SelectHealthSourceServices/SelectHealthSourceServices.tsx:32]: In most health source we show error below SLI checkbox, by setting SLI  checkbox as touched true , so we will need to change this behaviour.

check **submitData** function in src/modules/85-cv/pages/health-source/connectors/AppDynamics/AppDHealthSource.utils.ts

Comment (harshilbits) [reply src/modules/85-cv/pages/health-source/common/SelectHealthSourceServices/SelectHealthSourceServices.tsx:32]: Even if the field is touched , no behaviour will change. Its just that default SLI value is selected.

Comment (deepesh-ui) [reply src/modules/85-cv/pages/health-source/common/SelectHealthSourceServices/SelectHealthSourceServices.tsx:32]: The error will be shown below SLI checkbox  if the fields is marked touched and user submits form the very first time

**Review by deepesh-ui — APPROVED** [review]

**Comment by sunilgupta-harness** [comment]
cypress test is taking long time to finish

**Pull request merged**